backend/app/tasks/knowledge_tasks.py

```
@@ -0,0 +1,346 @@
+        user_name: Username
+    """
+    logger.info(
+        f"[Celery Summary] Task started: document_id={document_id}, user_id={user_id}"
+    )
+
+    db = SessionLocal()
+    try:
+        global_summary_enabled = getattr(settings, "SUMMARY_ENABLED", False)
+        if not global_summary_enabled:
+            logger.info(
+                f"[Celery Summary] Skipping: global summary not enabled"
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)